Resources/public/templates/confirmbox/draftconflict.hbt

```diff
@@ -0,0 +1,54 @@
+<div class="ez-draft-conflict-header">
+    <div class="ez-draft-conflict-header-subtitle">
```

Comment:
usually when you start adding a class with `header` it's a good sign that you should use the `header` tag and/or a `hX`